rackcorp/resourceRackcorpServer.go

```diff
@@ -167,7 +173,7 @@ func resourceRackcorpServer() *schema.Resource {
 				Sensitive: true,
 			},
 			"data_center_id": {
-				Type:     schema.TypeString,
+				Type:     schema.TypeInt,
```

Comment:
This was originally a string because we didn't want to couple to Rackcorp's choice of ID data type but keep it opaque.